### PR TITLE
Listening to readyToClose event and fire jitclose message

### PR DESCRIPTION
**Pull request by soichih**
I am still seeing some rooms having more people than there are. I *believe* what's happening is that there are people out there whose Jitsu connection is closed (maybe by pushing the hungup button) but the tab is left open. This update listens to readyToClose event and send `jitclose` to the WS server.

### CODE REVIEW

File: ui/config/index.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ module.exports = {
   },
   dev: {
     env: require('./dev.env'),
-    port: 8080,
+    port: 8090,
```
Comment (yarikoptic) [inline ui/config/index.js:26]: What is this port change about? Just curious

**Review by yarikoptic — COMMENTED** [review]
A question

File: ui/config/index.js (hunk 1 above)
Comment (soichih) [reply ui/config/index.js:26]: Ah.. it's not directly related to this PR, but  I had to update it so I can test the code. This port is used to run the local dev instance (by running.. npm run dev) and 8080 is the default port that many other localhost web servers use. I had another UI running on my machine using 8080, so I had to change the port for this app before I can test this PR. 

Sorry for another sloppy git commit..!!

Comment (yarikoptic) [reply ui/config/index.js:26]: Thank you for the explanation!
We are all yet to raise up to @kyleam 's standard in commits and PR composition ;-)

**Pull request merged**